### PR TITLE
DO NOT MERGE YET: disable auto-import. Allow one final import when going live

### PR DESCRIPTION
**Pull request by jwo**

### CODE REVIEW

**Review by ironcoconut — DISMISSED** [review]

**Comment by jwo** [comment]
I think we can wait for #32 to merge in before merging this... But this _can_ merge in once #32 is merged and this is up to date

**Comment by jwo** [comment]
Closing in favor of evaluating what code should live and die bespokely. 

#coderot

**Pull request closed without merging**
